_data/simple-icons.json

```diff
@@ -2636,6 +2636,11 @@
             "source": "https://designnavigator.daimler.com/Daimler_Corporate_Logotype_Black_DTP",
             "guidelines": "https://designnavigator.daimler.com/Daimler_Corporate_Logotype"
         },
+        {
+            "title": "Dapr",
+            "hex": "0D2192",
+            "source": "https://github.com/dapr/dapr/tree/master/img"
```

Comment:
```suggestion
            "source": "https://raw.githubusercontent.com/dapr/dapr/f5a5acc406302f0d5122ae30d18f9baba6dba8d3/img/dapr_logo.svg"
```

Reply:
Let's use permalink instead.

Reply:
> Let's use permalink instead.

Okay 👌

Reply:
> How do I reproduce this problem? (@helbing)

You could refer to https://github.com/simple-icons/simple-icons/pull/7752#issuecomment-1238557522.

Reply:
> 

thk